pantry-finder-api/app/controllers/api/foodbanks_controller.rb

```diff
@@ -11,6 +11,8 @@ def index
       render json: serialized_foodbanks
     end
 
+    private
```

Comment:
@PhilNorman2 While working on the deployment I learned that Jets deploys every public method in a controller as an individual lambda. Something to keep in mind as we add more endpoints and controllers.